### PR TITLE
migrate the wiki guide links to the community repo & small changes

### PR DESCRIPTION
**Pull request by KobeW50**

### CODE REVIEW

File: youtube/guide-for-beginners-lemmy.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Wait for the patching to be complete. It usually takes 2-5 minutes. Do not leave
 
 
 
-When patching is finished you can save the APK as a file so that you have it for later in case you have trouble installing it or want to share it. To do this, tap the three dots in the upper right corner of the screen and then press "Export APK" as shown in [this image](https://imgur.com/a/JqmfzAj).
+When patching is finished you can save the APK as a file so that you have it for later in case you have trouble installing it or want to share it. To do this, tap save icon in the lower left corner of the screen as shown [here](https://imgur.com/a/FKD0okE).
```
Comment (KobeW50) [inline youtube/guide-for-beginners-lemmy.md:84]: new way to export apks

File: youtube/guide-for-beginners.md [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
+* This guide is for non-root installs.
 
 
 
 * This guide will use the RVX Manager patching method. Check the [documentation](https://github.com/inotia00/revanced-documentation#revanced-extended-documentation) for info on the other patching methods.
 
 
 
-* Check the [YT troubleshooting page](https://www.reddit.com/r/revancedextended/wiki/yt-troubleshooting/#wiki_issues_with_patching_.26amp.3B_installation) if you get stuck during this guide.
+* Check this [YT troubleshooting page](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-troubleshooting.md#issues-with-patching--installation) if you get stuck during this guide.
```
Comment (KobeW50) [inline youtube/guide-for-beginners.md:43]: im migrating this stuff to the community repo

File: youtube/guide-for-beginners.md [hunk 3]
```diff
@@ -165,16 +165,16 @@ ___
...
 
 
 
-Read the [YT ReVanced Extended wiki guide](https://www.reddit.com/r/revancedextended/wiki/yt-guide/) for info on setting up a downloader.
+Read the "Downloader Setup" section in the [YT ReVanced Extended community guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-guide.md#downloader-setup) for info on setting up a downloader.
 
 
 
-Check out [this repository](https://github.com/kazimmt/RVX-Features#youtube-revanced-extended-features) to see what all of the YT ReVanced Extended settings do!
+Check out [this site](https://kazimmt.github.io/RVX-Features/rvx-features/yt-rvx-features/) to see what all of the YT ReVanced Extended settings do!
```
Comment (KobeW50) [inline youtube/guide-for-beginners.md:176]: he moved it from a repo to a nice site

**Comment by KobeW50** [comment]
and ofc remember to update the "Last updated by Spacellary on..." if u merge this

https://github.com/ReVanced-Extended-Community/Community-Guides/blob/12c32f8d2559792edeac8e8e875d29ba1846d49a/youtube/guide-for-beginners.md?plain=1#L18C1-L18C58

**Review by Spacellary — APPROVED** [review]
Nice, good changes

**Pull request merged**